### ISSUE TITLE
Picker has been extracted from react-native core - Warning

### ISSUE THREAD

**Issue by kalviapp** [wix/react-native-ui-lib#814]
'Warning: Picker has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from \'@react-native-community/picker\' instead of \'react-native\'. See https://github.com/react-native-community/react-native-picker'

```
info Fetching system and libraries information...
System:
    OS: macOS 10.15.5
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 2.07 GB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.19.0 - ~/.nvm/versions/node/v10.19.0/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v10.19.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
    Xcode: 11.5/11E608c - /usr/bin/xcodebuild
  Languages:
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1
    react-native: 0.62.2 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```

**Comment by ethanshar** [comment]
Duplicate of #788

**Comment by ethanshar** [comment]
This will be fixed in our next version (5.12.0)

**Comment by SanjanaTailor** [comment]
@ethanshar, Facing same issue in any other file has not mentioned Picker from React native.
Using import RNPickerSelect from 'react-native-picker-select';  from this npm package and facing same issue as Error warning  Warning: Picker has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/picker' instead of 'react-native'. See https://github.com/react-native-community/react-native-picker
RNPickerSelect@http://localhost:8081/index.bundle?platform=android&dev=true&minify=false:124424:36

**Comment by ethanshar** [comment]
We had just released a new version 5.12.0 that use the community/react-native-picker. 
Please update your RNUILib version 
Also make sure to install as a dependency `@react-native-community/picker`
